package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "workshop-protractor",
+  "version": "1.0.0",
+  "description": "This is a Workshop about Protractor",
+  "main": "index.js",
+  "scripts": {
+    "clean": "del -rf dist",
```

Comment:
Please work with a linux based console, for example, Git bash.  On Linux, the command to remove is rm not del.